src/main/java/io/fabric8/jenkins/openshiftsync/OpenShiftUtils.java

```diff
@@ -270,6 +271,14 @@ public static void cancelOpenShiftBuild(Build build) {
       .done();
   }
 
+  public static void updateOpenShiftBuildToPending(Build build) {
```

Comment:
Maybe you can create a generic function that takes the Phase to set it to...PENDING, CANCELLED, etc